app/assets/stylesheets/styles.scss

```diff
@@ -40,25 +46,4 @@ li {
 .side-panel-image {
   text-align: center;
   margin: 1.5em;
-}
-
-.favorite-button, .unfavorite-button {
```

Comment:
I don’t even remember touching that, sorry. I will add back in as soon as I can.

> On Feb 2, 2015, at 1:52 AM, Ryan Mitchell <notifications@github.com> wrote:
> 
> In app/assets/stylesheets/styles.scss <https://github.com/tiy-q1-ror-chs-2015/Team-1/pull/73#discussion_r23909219>:
> 
> > @@ -40,25 +46,4 @@ li {
> >  .side-panel-image {
> >    text-align: center;
> >    margin: 1.5em;
> > -}
> > -
> > -.favorite-button, .unfavorite-button {
> These styles are used on the School#show page for the heart icon. Would you add them back in or modify them (as you see fit)?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/tiy-q1-ror-chs-2015/Team-1/pull/73/files#r23909219>.
>